---
layout: default
title: ekviky Projects 
---


	<div class="projects">
		<h1>Projects</h1>
		<p>My main research interest is to understand the impact of structural variation (insertions, deletions, transposable elements, large rearrangements, etc.) on genomes and phenotypes. Below I outline previous and ongoing projects focusing on the tricky bits of the genome - nuisance events, junk DNA, inaccessible regions - and provide pointers to open-source bioinformatics tools and collaborations.<p>
		<ol>
			<h2><li>TEs</li></h2>
			<p>Transposable elements (TEs) are sequences that can transpose or "jump" from position to position within genomes. TE insertions have contributed to the sequence divergence between species, diversity within populations, as well as phenotypes like genetic disease and cancers. Not only are TEs ubiquitous in the human genome, their <a href="http://genome.cshlp.org/content/20/5/600.long">accumulation is distributed non-randomly</a> and local sequence characteristics - or landscape features - can tell a lot about their insertion mechanisms and the forces shaping their fixation over evolutionary time.<p>
			<p>TE mobility can also potentially have a great impact on gene expression (by disrupting exons, ablating splice signals, donating of novel transcription start sites) which could lead to genetic disease. Yet their contribution to human disease is poorly understood. This is largely due to the challenges of sequencing specifically the active copies in a genome. With <a href="http://www.well.ox.ac.uk/gerton-lunter#research">Gerton Lunter</a> and colleagues at the Wellcome Trust Centre for Human Genetics, I have developed <a href="https://www.biorxiv.org/content/early/2017/09/22/192708">high-throughput molecular and computational genomics approaches</a> to detect TEs in whole genome sequencing (WGS) data. The protocol and tools to detect polymorphic and de novo insertions are available on <a href="https://ekviky.github.io/TE-NGS/">github</a>.<p>
			<p>In collaboration with clinical researchers at the Broad Institute and ExAC consortium, we have applied these tools to individual genomes with clinical phenotypes spanning a wide range of disorders. We hope to help find potential causal variants in these unsolved cases and illuminate the contribution of TEs to human disease.<p> 
			<h2><li>Small indels</li></h2>
			<p>Insertions and deletions (collectively "indels") of multiple nucleotides have played a major role in the evolution of genomes. Not just nuisance events, we now recognize that indels have shaped and been shaped by regional differences in <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030176">genomic landscape features</a>.<p>
			<p>Indel mutation rates are also <a href="http://genome.cshlp.org/content/19/7/1153.long">context dependent</a> and highly variable. Patterns of indel mutation and fixation dynamics provide clues to <a href="https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/mst185">cryptic indel hotspots</a> that can even mimic signatures of natural selection.<p>
			<p>After point mutations, indels are the second most common underlying source of individual variation among and between populations. Their <a href="http://genome.cshlp.org/content/23/5/749.long">population frequencies</a> have been influenced by selective constraint, revealing the underlying genomic functions of regions such as genes, and potentially functional non-coding conserved regions that impact individual phenotypes.<p> 
			<p>Given their importance to genome evolution, population-scale variation, and function, reliable and efficient identification of indels from genomic sequence data is crucial. With <a href="http://www.bx.psu.edu/makova_lab/">Kateryna Makova</a> at Penn State University, we developed <a href="https://usegalaxy.org/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/indels_3way/indels_3way/1.0.3">software</a> to identify and polarize indel mutations in multiple-species alignments.<p> 
			<h2><li>Viruses</li></h2>
			<p>Similar to TEs, viruses can insert themselves into the genomic sequence of their host. Indeed, viruses are thought to be ancient precursors of some TEs. Integration can be an important step in the viral life cycle, for example the mechanism of HPV-associated cancers (nearly all cervical, and a majority of oralpharangeal) depends upon it. However, questions such as how often, how many, where, and when remain unanswered. Viral integrations detected in WGS data collected from clinical samples and tissues can help to fill in the remaining gaps in our knowledge of viral dynamics in affected and healthy individual genomes.<p>
			<h2><li>Clinical NGS tools</li></h2>
			<p>Some scripts and pipelines useful for clinical WGS<p>
		</ol>
		<h2>Publications</h2>
		<ul class="pubs">
			<li><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Kvikstad+E">
				<i class="ai ai-pubmed-square" ai-3x aria-hidden="true"></i> Pubmed
			</a></li>
			<li><a href="https://scholar.google.co.uk/citations?user=uMVdpNsAAAAJ&hl=en&oi=ao">
				<i class="ai ai-google-scholar-square" ai-3x aria-hidden="true"></i> Google Scholar
			</a></li>
			<li><a href="https://orcid.org/0000-0002-2730-4472" target="orcid.widget" rel="noopener noreferrer" style="vertical-align:top;"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em;margin-right:.5em;" alt="ORCID iD icon">orcid.org/0000-0002-2730-4472</a></li>
			</ul>
	</div>
