---
layout: project
title: ekviky indels
permalink: /projects/indels/
---
# Small Indels
After point mutations, insertions and deletions (collectively "indels") of multiple nucleotides are the second most common source of individual variation among and between populations. Until relatively recently, however, indels have been largely ignored as "nuisance events" in analysis of DNA and amino acid sequences.  

Given their importance to genome evolution, population-scale variation, and function, reliable and efficient identification of indels from genomic sequence data is crucial. With [Kateryna Makova](http://www.bx.psu.edu/makova_lab/) at Penn State University, we developed [software](https://usegalaxy.org/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/indels_3way/indels_3way/1.0.3) to identify and polarize indel mutations in multiple-species alignments.   

Applying these tools to primate phylogeny, we now recognize that indels have shaped and been shaped by regional differences in [genomic landscape features](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030176).  

Indel mutation rates are also [context dependent](http://genome.cshlp.org/content/19/7/1153.long) and highly variable, in non-coding and non-repetitive DNA.  

Availability of population-scale data in 1000 Genomes Project helped us to demonstrate that indel [population frequencies](http://genome.cshlp.org/content/23/5/749.long) have been influenced by selective constraint, revealing the underlying genomic functions of regions such as genes, and potentially functional non-coding conserved regions that impact individual phenotypes.  

Patterns of indel mutation and fixation dynamics provide clues to [cryptic indel hotspots](https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/mst185) that can even mimic signatures of natural selection.  
