---
layout: default
title: ekviky indels
permalink: /projects/indels/
---
# Small Indels
Insertions and deletions (collectively "indels") of multiple nucleotides have played a major role in the evolution of genomes. Not just nuisance events, we now recognize that indels have shaped and been shaped by regional differences in [genomic landscape features](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030176).  

Indel mutation rates are also [context dependent](http://genome.cshlp.org/content/19/7/1153.long) and highly variable. Patterns of indel mutation and fixation dynamics provide clues to [cryptic indel hotspots](https://academic.oup.com/mbe/article-lookup/doi/10.1093/molbev/mst185) that can even mimic signatures of natural selection.  

After point mutations, indels are the second most common underlying source of individual variation among and between populations. Their [population frequencies](http://genome.cshlp.org/content/23/5/749.long) have been influenced by selective constraint, revealing the underlying genomic functions of regions such as genes, and potentially functional non-coding conserved regions that impact individual phenotypes.  

Given their importance to genome evolution, population-scale variation, and function, reliable and efficient identification of indels from genomic sequence data is crucial. With [Kateryna Makova](http://www.bx.psu.edu/makova_lab/) at Penn State University, we developed [software](https://usegalaxy.org/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/indels_3way/indels_3way/1.0.3) to identify and polarize indel mutations in multiple-species alignments.
