---
layout: project
title: ekviky TEs 
permalink: /projects/TEs/
---
# TEs

Transposable elements (TEs) are sequences that can transpose or "jump" from position to position within genomes. While most insertions are inactive relics and dismissed as "junk DNA", TE mobility has contributed to the sequence divergence between species, diversity within populations, as well as phenotypes like genetic disease and cancers. Not only are TEs ubiquitous in the human genome, their [accumulation is distributed non-randomly](http://genome.cshlp.org/content/20/5/600.long) and local sequence characteristics - or landscape features - can tell a lot about their insertion mechanisms and the forces shaping their fixation over evolutionary time.  

TE mobility can also potentially have a great impact on gene expression (by disrupting exons, ablating splice signals, donating novel transcription start sites) which could lead to genetic disease. Yet their contribution to human disease is poorly understood. This is largely due to the challenges of sequencing specifically the active copies in a genome. With colleagues at the Wellcome Trust Centre for Human Genetics, I have developed [molecular and computational genomics approaches](https://www.biorxiv.org/content/early/2017/09/22/192708) to detect TEs in whole genome sequencing (WGS) data. The protocol and tools to detect polymorphic and de novo insertions are available on [github](https://ekviky.github.io/TE-NGS/).  

In collaboration with clinical researchers at the Broad Institute and ExAC consortium, we have applied these tools to individual genomes with clinical phenotypes spanning a wide range of disorders. We hope to help find potential causal variants in these unsolved cases and illuminate the contribution of TEs to human disease.
